packages/donations-app/app/components/common/directives.js

```diff
@@ -329,13 +329,13 @@ angular.module('aliceApp')
       },
       /*jshint multistr: true */
       template: '<div >\
-                  <div class="btn btn-sm btn-primary"\
-                      style="height:34px; padding-top:7px; margin-top: 3px; text-align:center; width:10vw" ngf-select\
+                  <button\
+                      style="width:10vw !important" ngf-select\
```

Comment:
could we move it to scss or css?

Reply:
We have inline styles in different directives in this file. Maybe we should create additional PR to refactor it?